### DISCUSSION TITLE
Send Only suricata to external

### DISCUSSION THREAD

**Discussion by darth-drew** [Security-Onion-Solutions/securityonion#3513] [Unsupported Versions]
Greetings, I know this question has been touched on in a few discussions, but I wanted to see if we could bring a consolidated  answer in to this one regarding a specific issue.

I am trying to send suricata events to a remote server following this documentation:
https://docs.securityonion.net/en/latest/logstash.html#adding-new-logs

Thanks to you all, I am successful (YAY), but I am also seeing a large amount of zeek and ossec data on the external server, which is far from preferable in this case (Aww). I know this is a bit more of an elastic stack question, but am hopeful others will find this discussion helpful. My question: Is there a way to adapt these configs to send only suricata events to an external server, or is updating the global config the wrong approach?

Distributed 2.3.30 grid.
Global config additions (You'll see two logstash chunks here, I have tried uncommenting each individual chunk but WRT my question the behavior is the same):
![image](https://user-images.githubusercontent.com/12967094/111534451-6197ea00-873e-11eb-9cbf-d18797cf0971.png)
Custom config
![image](https://user-images.githubusercontent.com/12967094/111534584-8b511100-873e-11eb-9653-90f5b92e2e1e.png)

Looking forward to figuring this out! It is one of my last requirements to meet before making my adversaries cry..

Thanks

**Comment by weslambert** [comment]
Please see the following: https://docs.securityonion.net/en/latest/logstash.html?highlight=forwarding#forwarding-events-to-an-external-destination.

**Reply by darth-drew** [reply]
Sorry for the late reply. That did do the trick, Thanks! However, (you probably realized this already) the cloned events are also showing up in my SO kibana, meaning that I get two events for each time suricata fires. So if suricata fires 50 times, I see 50 events on my external destination, and 100 events in the kibana interface. Short of filtering out the "clone" tags from the queries in kibana, is there a way to ignore any "clone" events before they get there?

**Reply by weslambert** [reply]
I wouldn't expect that you would see duplicate events in Kibana, given that each event should only traverse one output.  Could you provide a redacted example?

**Comment by darth-drew** [comment]
![image](https://user-images.githubusercontent.com/12967094/113050499-e50bff00-9172-11eb-876c-dde02cb3ae1f.png)

**Comment by darth-drew** [comment]
Late update, but we were able to overcome this issue by using the mutate plugin, and not the clone filter plugin.
